config/ngrok/config.yml

```
@@ -1,9 +1,12 @@
 version: "3"
 
+agent:
+    authtoken: YOUR_AUTH_TOKEN
+
 tunnels:
   proxy1:
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)